### PR TITLE
LPS-195927 

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**
https://issues.liferay.com/browse/LPS-195927

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by david-gutierrez-mesa** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/applicationdisplaytemplate/ApplicationDisplayTemplatesWithStaging.testcase [hunk 1]
```diff
@@ -4,7 +4,7 @@ definition {
 	property ci.retries.disabled = "true";
 	property portal.release = "true";
 	property portal.upstream = "true";
-	property testray.main.component.name = "Product Menu";
+	property testray.main.component.name = "Widget Page Templates";
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/applicationdisplaytemplate/ApplicationDisplayTemplatesWithStaging.testcase:7]: Should be `Templates`, Application Display Template is Widget Templates

**Review by Tim-Cao — COMMENTED** [review]
@david-gutierrez-mesa

**Comment by Tim-Cao** [comment]
ci:stop

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/applicationdisplaytemplate/ApplicationDisplayTemplatesWithStaging.testcase (hunk 1 above)
Comment (david-gutierrez-mesa) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/applicationdisplaytemplate/ApplicationDisplayTemplatesWithStaging.testcase:7]: true

**Comment by david-gutierrez-mesa** [comment]
ci:close

**Comment by david-gutierrez-mesa** [comment]
see https://github.com/liferay-echo/liferay-portal/pull/14504

**Pull request closed without merging**